### DISCUSSION TITLE
1.5.1.2

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#556] [Обновления]
## Список изменений
- Исправлен баг с работой на YouTube Embed
- Изменена логика поиска плеера на Twitch

Все изменения были внесены @SashaXser в https://github.com/ilyhalight/voice-over-translation/pull/555

## What's Changed
* 1.5.1.2 by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/555

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.1.1...1.5.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.1.2'>1.5.1.2</a>.</em>

**Comment by NullVerdict** [comment]
@ilyhalight Стоит ли скрывать кнопку перевода если не найден айди видео?

**Reply by ilyhalight** [reply]
> Стоит ли скрывать кнопку перевода если не найден айди видео?

Думаю можно скрывать

**Reply by NullVerdict** [reply]
Можно же обновление перезалить? Изменение не очень значительное.

**Comment by FiLinX** [comment]
урряя!!  эмбед зер гуд!  ))
и чё странно - настройки уровней шкал громкости подтянулись из непонять откуда)
ну, я когда пробовал 1.5.1, то сначала в темпермонке я отключал 1.4.1 а затем и удалял его, и только после ставил 1.5.1
и естесственно мне пришлось делать настройки шкал в панельке заново
потом я накатывал обновой 1.5.1.1
проверив неработоспособность на ют-эмбедах там, я взял и просто в темпермонке в утилитах сделал "Импорт из файла" его бекапа ещё с 1.4.1
а сейчас же я снова накатил 1.5.1.2 обновлением  -  и опа, настройки шкал оказались уже выставленными как я делал в 1.5.1
походу что при накатке бекапа темпермонки с 1.4.1 поверх 1.5.1.1 (не удалив его)  -  настройки от этого 1.5.1.1 всё же где то в компе остались, видать не удаляются они при импорте бекапа темпермонки

**Comment by FiLinX** [comment]
ребят, а как должна работать функция "связать громкость видео и перевода" ?
я думал, что при выставленной этой настройке, когда я запустив видео слушаю в переводе и регулирую ютубную громкость - то при этом должна синхронно изменяться и громкость перевода  -  но всё не так  -  изменяется только громкость оригинала аудио..

**Comment by FiLinX** [comment]
аа, всё, понял!
та установка на связывание  -  она действует только если регулировать на ползунках в самой панельке!
